"more diff" vs "full diff"
--- pytest-mock-3.10.0/tests/test_pytest_mock.py.orig	2022-10-05 20:52:25.000000000 +0200
+++ pytest-mock-3.10.0/tests/test_pytest_mock.py	2022-10-24 06:26:54.219495582 +0200
@@ -773,12 +773,12 @@ def test_detailed_introspection(testdir:
         "*Args:",
         "*assert ('fo',) == ('',)",
         "*At index 0 diff: 'fo' != ''*",
-        "*Use -v to get more diff*",
+        "*Use -v to get * diff*",
         "*Kwargs:*",
         "*assert {} == {'bar': 4}*",
         "*Right contains* more item*",
         "*{'bar': 4}*",
-        "*Use -v to get more diff*",
+        "*Use -v to get * diff*",
     ]
     result.stdout.fnmatch_lines(expected_lines)
 
@@ -814,12 +814,12 @@ def test_detailed_introspection_async(te
         "*Args:",
         "*assert ('fo',) == ('',)",
         "*At index 0 diff: 'fo' != ''*",
-        "*Use -v to get more diff*",
+        "*Use -v to get * diff*",
         "*Kwargs:*",
         "*assert {} == {'bar': 4}*",
         "*Right contains* more item*",
         "*{'bar': 4}*",
-        "*Use -v to get more diff*",
+        "*Use -v to get * diff*",
     ]
     result.stdout.fnmatch_lines(expected_lines)
 
